Gemfile

```diff
@@ -26,7 +26,7 @@ gem 'azure_mgmt_dns',                       path: 'management/azure_mgmt_dns'
 gem 'azure_mgmt_event_hub',                 path: 'management/azure_mgmt_event_hub'
 gem 'azure_mgmt_features',                  path: 'management/azure_mgmt_features'
 gem 'azure_mgmt_graph',                     path: 'management/azure_mgmt_graph'
-gem 'azure_mgmt_insights',                  path: 'management/azure_mgmt_insights'
+gem 'azure_mgmt_monitor',                   path: 'management/azure_mgmt_monitor'
```

Comment:
Sure, marking the insights gem as deprecated may be fine. We should definitely include this in our "change log" and "release notes" (if we include any).